jumanji/environments/packing/bin_pack/env.py

```diff
@@ -535,7 +543,12 @@ def is_action_allowed(
         )(obs_ems, obs_ems_mask, items, items_mask, items_placed)
         return action_mask
 
-    def _pack_item(self, state: State, ems_id: int, item_id: chex.Numeric) -> State:
+    def _pack_item(
```

Comment:
Was this reformatted automatically by your IDE? Might be worth putting it back to what it was if possible (It doesn't look like anything has functionally changed)

Reply:
Done ✅